### ISSUE TITLE
[Suggestion] Hepta windows

### ISSUE THREAD

**Issue by PopkaAkula** [tsunamayo/Skywanderers#1743]
### Short Description of the suggestion:

I've recently built a cockpit and I've come across a problem.

![image](https://user-images.githubusercontent.com/37738021/79081411-6fb6dc80-7d1d-11ea-8e15-c5204571c70a.png)

When making these kinds of shapes, a window hepta is required because the penta sticks out a bit too much.

![image](https://user-images.githubusercontent.com/37738021/79081434-a1c83e80-7d1d-11ea-9963-5b4165b72cb4.png)

It ruins the sleekness of it and I would bet that a lot of people would like to have this kind of shape.

**Comment by Aiwii** [comment]
A few of my ships use heptas actually, I'm surprised that they are missing.

**Comment by tsunamayo** [comment]
Yes indeed it makes sense. I think it might post ea though, because adding new shape is time consuming because of collisions... 
I want to find a way to speed up the process of adding shape for that reason, and also for modding support.
Thanks!
